code/modules/fish/fishtank.dm

```diff
@@ -112,7 +112,7 @@
 	if(light_switch)
 		set_light(2,2,"#a0a080")
 	else
-		set_light(0)
+		adjust_light()
```

Comment:
While I did check and we don't have a proc named adjust_light in the lighting system, I'll rename this to help avoid that confusion.